### PR TITLE
Release Notes for 202405

### PR DESCRIPTION
**Pull request by prasanna228**
Release Notes for 202405 build

### CODE REVIEW

**Comment by zhangyanzhao** [comment]
@bingwang-ms  @liat-grozovik  @adyeung @saiarcot895  can you please help to review this PR and double check the SAI, kernel  update etc? Thanks.

File: doc/release-notes/SONiC_202405_Release_Notes.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+# Dependency Version
+
+|Feature                    | Version  |
+| ------------------------- | --------------- |
+| Linux kernel version      | linux_6.1.38-4  |
+| SAI   version             | SAI v1.14.0    |
+| FRR                       | 8.5.4   |
+| LLDPD                     | 1.0.16-1+deb12u1 |
+| TeamD                     | 1.30-1    |
```
Comment (saiarcot895) [inline doc/release-notes/SONiC_202405_Release_Notes.md:30]: teamd is now on 1.31-1.

File: doc/release-notes/SONiC_202405_Release_Notes.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+| SNMPD                     | 5.9+dfsg-4+deb11u1    |
+| Python                    | 3.11.2-6    |
+| syncd                     | 1.0.0    |
+| swss                      | 1.0.0    |
+| radvd                     | 2.18-3    |
+| isc-dhcp                  | 4.4.3-P1-2  |
+| sonic-telemetry           | 1.1    |
+| redis-server/ redis-tools | 5.0.3-3~bpo9+2    |
```
Comment (saiarcot895) [inline doc/release-notes/SONiC_202405_Release_Notes.md:38]: Redis is on 7.0.15-1~deb12u1

File: doc/release-notes/SONiC_202405_Release_Notes.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+| Linux kernel version      | linux_6.1.38-4  |
+| SAI   version             | SAI v1.14.0    |
+| FRR                       | 8.5.4   |
+| LLDPD                     | 1.0.16-1+deb12u1 |
+| TeamD                     | 1.30-1    |
+| SNMPD                     | 5.9+dfsg-4+deb11u1    |
+| Python                    | 3.11.2-6    |
+| syncd                     | 1.0.0    |
+| swss                      | 1.0.0    |
+| radvd                     | 2.18-3    |
```
Comment (saiarcot895) [inline doc/release-notes/SONiC_202405_Release_Notes.md:35]: radvd is on 2.19-1+b1

File: doc/release-notes/SONiC_202405_Release_Notes.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+# Dependency Version
+
+|Feature                    | Version  |
+| ------------------------- | --------------- |
+| Linux kernel version      | linux_6.1.38-4  |
+| SAI   version             | SAI v1.14.0    |
+| FRR                       | 8.5.4   |
+| LLDPD                     | 1.0.16-1+deb12u1 |
+| TeamD                     | 1.30-1    |
+| SNMPD                     | 5.9+dfsg-4+deb11u1    |
```
Comment (saiarcot895) [inline doc/release-notes/SONiC_202405_Release_Notes.md:31]: snmpd is on 5.9.3+dfsg-2

File: doc/release-notes/SONiC_202405_Release_Notes.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+Note : The kernel version is migrated to the version that is mentioned in the first row in the above 'Dependency Version' table.
+
+
+# Security Updates
+
+1. Kernel upgraded from 5.10.136-1 to 6.1.38-4 for SONiC release.<br>
+   Change log: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.38
+
+2. Docker upgraded from  24.0.7-debian-stretch to 27.0.1-debian-stretch <br>
```
Comment (saiarcot895) [inline doc/release-notes/SONiC_202405_Release_Notes.md:49]: Where is this coming from? The docker daemon is still on 24.0.2-1~debian.12~bookworm

File: doc/release-notes/SONiC_202405_Release_Notes.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+| sonic-telemetry           | 1.1    |
+| redis-server/ redis-tools | 5.0.3-3~bpo9+2    |
+| Debian version			| Migrated to Bookworm (Debian version 12)	|
+
+Note : The kernel version is migrated to the version that is mentioned in the first row in the above 'Dependency Version' table.
+
+
+# Security Updates
+
+1. Kernel upgraded from 5.10.136-1 to 6.1.38-4 for SONiC release.<br>
```
Comment (saiarcot895) [inline doc/release-notes/SONiC_202405_Release_Notes.md:46]: The kernel was previously on 5.10.179.

File: doc/release-notes/SONiC_202405_Release_Notes.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+| ***Add SRv6 SID L3Adj*** | This feature describes the extensions of SRv6Orch required to support the programming of the L3Adj associated with SRv6 uA, End.X, uDX4, uDX6, End.DX4, and End.DX6 behaviors.     |  [1472](https://github.com/sonic-net/SONiC/pull/1472)     | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
+| ***Base OS upgrade to Bookworm*** | This feature upgrades SONiC's base image from Debian Bullseye to Debian Bookworm with the notable changes. | [17234](https://github.com/sonic-net/sonic-buildimage/pull/17234)  | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
+| ***Bookworm Upgrade LLDP, SNMP subagent, ICCPD, PDE*** | This feature implements the bookworm upgrade for LLDP, SNMP subagent, ICCPD, PDE |  [1677](https://github.com/sonic-net/SONiC/issues/1677)     | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
```
Comment (adyeung) [inline doc/release-notes/SONiC_202405_Release_Notes.md:65]: The set of code PR also includes FRR bookworm migration, pls modify to

 | ***Bookworm Upgrade LLDP, SNMP subagent, ICCPD, PDE, FRR*** | This feature implements the bookworm upgrade for LLDP, SNMP subagent, ICCPD, PDE, FRR | [1677](https://github.com/sonic-net/SONiC/issues/1677) | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |

File: doc/release-notes/SONiC_202405_Release_Notes.md (hunk 1 above)
Comment (prasanna228) [reply doc/release-notes/SONiC_202405_Release_Notes.md]: Addressed as suggested.

File: doc/release-notes/SONiC_202405_Release_Notes.md (hunk 2 above)
Comment (prasanna228) [reply doc/release-notes/SONiC_202405_Release_Notes.md]: Addressed as suggested.

File: doc/release-notes/SONiC_202405_Release_Notes.md (hunk 3 above)
Comment (prasanna228) [reply doc/release-notes/SONiC_202405_Release_Notes.md]: Addressed as suggested.

File: doc/release-notes/SONiC_202405_Release_Notes.md (hunk 4 above)
Comment (prasanna228) [reply doc/release-notes/SONiC_202405_Release_Notes.md]: Addressed as suggested.

File: doc/release-notes/SONiC_202405_Release_Notes.md (hunk 5 above)
Comment (prasanna228) [reply doc/release-notes/SONiC_202405_Release_Notes.md]: Addressed as suggested.

File: doc/release-notes/SONiC_202405_Release_Notes.md (hunk 6 above)
Comment (prasanna228) [reply doc/release-notes/SONiC_202405_Release_Notes.md]: Addressed as suggested.

File: doc/release-notes/SONiC_202405_Release_Notes.md (hunk 7 above)
Comment (prasanna228) [reply doc/release-notes/SONiC_202405_Release_Notes.md]: Updated as suggested.

**Review by adyeung — APPROVED** [review]

**Pull request merged**